src/models/mod.rs

```diff
@@ -125,7 +125,7 @@ pub(crate) async fn start_after_interruption(
     pool: &sqlx::Pool<sqlx::Postgres>,
 ) -> anyhow::Result<u64> {
     let query = "SELECT block_height
```

Comment:
Let's say here something like "block_height - 1000"
Now we are collecting the blocks in one thread, but there's an easy way to parallelise this, and it may give us the gaps that we should fill correctly.
Since postgres anyway cares about avoiding duplicates, it's much safer just to re-run this range again.

Be careful, this code can be used in localnet, and you can have negative block_height
You need to take 0 as the default min value